### PR TITLE
Vespa-App

### PR DESCRIPTION
**Pull request by looh16**
### Note: You need to create a master.key in backend repo after cloning the project and add the key given.
**For this project the following functionalities were created:**

1. login endpoint
2. create account endpoint
3. logout endpoint
4. get all products endpoint
5. create products endpoint
6. delete products endpoint
7. create reservations endpoint
8. delete reservations endpoint

[Frontend Link](https://github.com/ishpaul777/vespa-clone-frontend)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,115 @@
-# vespa-clone-backend
\ No newline at end of file
+<a name="readme-top"></a>
```
Comment (Meltrust) [inline README.md:1]: - [ ] On your readme, kindly add a "Tests" section with specific instructions about how to run your tests.  That way, your app will be well-documented. 💪 

- [ ] There are two files with tests not yet implemented:

image

These are reservations_helper_spec.rb and reservation_spec.rb.   Kindly either add some tests on those files or remove them.

- [ ]  You are using Rspec, but the "test" folder still exists, which is used by another test framework. Kindly consider deleting that folder if you are not using it. 👍

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @looh16,

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation, but you are almost there!






### To highlight:

-  Tests are passing✔️
-  API is working well✔️ 
-  Good readme ✔️ 




#### You are really close to finishing the Microverse program!! Keep it up! 👍👍👍

You can do it

### After implementing the requested changes, please submit another review request. ♻️
_Check the comments under the review._

_I strongly recommend you take them into account as they can improve your final evaluation._





Cheers and Happy coding!👏👏👏

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the previous reviews unless it is requested otherwise._**

------

**Review by vichuge — COMMENTED** [review]
Hi dear student, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

star_wars_almost_there

## Suggested changes
_Check the comments under the review._

## Everything is excellent here (back-end), but there is one bug to fix on the front so, good luck!

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by looh16** [comment]
Thanks!

**Review by Meltrust — APPROVED** [review]
#### Hi @looh16,


# **Wow, you did it 🎉**
Brilliant
## Thank you for the changes implemented 💪 🥇 ㊗️ 
#### _Unless you want to add more features, go ahead to your final presentation ⏩ ⏩ ⏩_ 
## You are about to finish the Microverse program.  You have come a long way!!!




### Good luck in the software industry!!  I'll see you there. ✨
### **Congratulations!!!!!!**  🎉
applause

### To highlight

- Excellent job ✔️ 




Cheers and Happy coding!👏👏👏


------

**Pull request merged**